PxWeb/Code/PxDatabase/Builders/MenuBuilder.cs

```diff
@@ -478,11 +478,7 @@ private static string GetNames(PCAxis.Paxiom.Variable variable)
                 for (int i = 0; i < 4; i++)
                 {
                     names.Append(variable.Values[i].Value);
-                    if (i < 5 || i == variable.Values.Count)
-                    {
-                        names.Append(", ");
-                    }
-
+                    names.Append(", ");
```

Comment:
Do we really need all the values in the Menu.xml? The file could become very large.

Reply:
No, the result is exactly as before. It was just SonarCube that spotted an always true block.